From 39a556c9ea9bf384b74ab3934441f860e424b5f1 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 1 Jul 2019 22:24:36 +0300
Subject: [PATCH 11/12] cl-som-imx8: defconfig: Enable USB ethernet

Added:
CONFIG_USB_HOST_ETHER=y
CONFIG_USB_ETHER_ASIX88179=y
CONFIG_USB_ETHER_MCS7830=y

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 configs/cl-som-imx8_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/cl-som-imx8_defconfig b/configs/cl-som-imx8_defconfig
index 5553f70..7a7911c 100644
--- a/configs/cl-som-imx8_defconfig
+++ b/configs/cl-som-imx8_defconfig
@@ -57,5 +57,8 @@ CONFIG_USB_GADGET_VENDOR_NUM=0x0525
 CONFIG_USB_GADGET_PRODUCT_NUM=0xa4a5
 CONFIG_SDP_LOADADDR=0x40400000
 CONFIG_USB_GADGET_DOWNLOAD=y
+CONFIG_USB_HOST_ETHER=y
+CONFIG_USB_ETHER_ASIX88179=y
+CONFIG_USB_ETHER_MCS7830=y
 CONFIG_VIDEO=y
 CONFIG_VIDEO_IMX8_HDMI=y
-- 
1.9.1

